platform/broadcom/saibcm-modules/debian/opennsl-modules.init

```diff
@@ -53,17 +53,12 @@ function load_kernel_modules()
     modprobe linux-kernel-bde dmasize=$dmasize maxpayload=128 debug=4 dma_debug=1 usemsi=$usemsi
     modprobe linux-user-bde
 
-    # Using insmod with absolute path for psample to make sure bcm psample is loaded.
-    # There is a different psample.ko module getting created at net/psample/psample.ko 
-    insmod /lib/modules/$(uname -r)/extra/psample.ko
```

Comment:
This is required for sflow support. We may want to ask BCM for providing this.

Reply:
These files are absent in 6.5.19 gpl modules. Will it work if psample related files are added back or should BRCM give us the psample code for 6.5.19.

Reply:
It was added as part of 3.7 and with BUILD_PSAMPLE=1 (locally built driver as part of opennsl). We may want to get this clarified with Broadcom on what psample driver to be used.